### DISCUSSION TITLE
How to get student developer pack if date is on the back side of the id card

### DISCUSSION THREAD

**Discussion by saksham60** [community/community#60441] [GitHub Education]
### Hi everyone,

As a full-time student at the Indian Institute of Technology, Madras, I am currently engaged in an off-campus project that has placed me at a remote location away from the institute. Consequently, this has led to certain complications in providing the necessary verification documentation.

My identification card, which serves as a standard proof for students, unfortunately, does not display the date on the front side. This has been problematic since the date is required for verification purposes. Furthermore, I am unable to provide a 'No Dues' form, as my current location prevents me from physically accessing the campus.

For these reasons, I am reaching out to seek guidance on how to navigate this situation. I assure you that I am fully prepared to submit any additional verification documents as necessary via email, should this method be accepted.

On another note, I am also encountering difficulties with GitHub, as the platform only permits screenshot submission. Specifically, I am having trouble resolving an issue related to location identification on the platform. I would greatly appreciate any support or direction you could offer to help rectify this issue.

Thank you for your time and attention to this matter. I eagerly await your response, and I am hopeful for a suitable solution soon.

**Comment by GhostOf0days** [comment]
I personally sent a screenshot of my cost of attendance bill. It must have your school logo on it, however.

**Comment by saksham60** [comment]
Can you please share me some email / link where i can send them personally.
I have school logo on my student portal that might work
Thanks in advance

**Reply by GhostOf0days** [reply]
A student portal won’t work. Like I said, try a screenshot of your cost of attendance bill with your school logo on it. Upload it [where you apply for the student developer pack benefits](https://education.github.com/benefits?type=student).

**Comment by jshtuazon2** [comment]
01-251412641-9

**Comment by saksham60** [comment]
Yes I did that but got 

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘Other’ option when submitting a GitHub Education support ticket.



I m not near campus

**Reply by GhostOf0days** [reply]
It seems like you have to be near campus then. Try again when you're close to it. They probably check your IP.

**Comment by samus-aran** [comment]
Hi @saksham60 

Regarding to the Student Dated ID we would suggest that you take a photo of another piece of evidence along side your Student ID card that has your date of course or session. Yes we do prefer students to be on campus when applying but we do accept students that are deemed far from campus as well.

Hope this helps, let us know how you get on 👍

**Comment by bahadirdogru** [comment]
Also my date information on backside of my ID card. And they not accept merged image or uploading 2 images.
